tested/dodona.py

```diff
@@ -54,6 +54,7 @@ class Status(str, Enum):
     WRONG = "wrong"
     INTERNAL_ERROR = "internal error"
     OUTPUT_LIMIT_EXCEEDED = "output limit exceeded"
+    NOT_EXECUTED = "wrong"
```

Comment:
I would insert a reference to the relevant issue in Dodona here, since this package is supposed to represent the Dodona format.